src/components/app/routes/loaders/rootLoader.js

```diff
@@ -264,10 +278,21 @@ const fetchEnterpriseCuration = async (enterpriseUUID, options = {}) => {
     ...options,
   });
   const url = `${getConfig().ENTERPRISE_CATALOG_API_BASE_URL}/api/v1/enterprise-curations/?${queryParams.toString()}`;
-  const response = await getAuthenticatedHttpClient().get(url);
-  const data = camelCaseObject(response.data);
-  // Return first result, given that there should only be one result, if any.
-  return data.results[0];
+
+  try {
+    const response = await getAuthenticatedHttpClient().get(url);
+    const data = camelCaseObject(response.data);
+    // Return first result, given that there should only be one result, if any.
+    if (data.results.length > 0) {
+      return data.results[0];
+    }
```

Comment:
nit: likely don't need  this conditional given `return data.results[0];` will not throw a JS error even if `data.results` is an empty array.